torch/_subclasses/fake_tensor.py

```diff
@@ -773,6 +773,15 @@ def __init__(
     def __torch_dispatch__(self, func, types, args=(), kwargs=None):
         kwargs = kwargs if kwargs else {}
 
+        from torch._decomp import decomposition_table
+
+        if func in decomposition_table:
```

Comment:
this shouldnt be before the constants handling, because we might need to invalidate constants

Reply:
Its just jammed in there for demonstration purposes, It probably needs to be called a good deal lower!